sdk/util.js

```diff
@@ -11,7 +11,9 @@ const WS_PROXY_URL = "ws://nft-sales-websocket.dappradar.com";
 const HTTP_PROXY_URL = "http://nft-sales-websocket.dappradar.com";
 
 const options = {
-    headers,
+    timeout: 1000,
+    // We had to adjust here as the http lib expects an array value (see node_modules/web3-providers-http/src/index.js, line 79 -> .forEach() function)
```

Comment:
In my opinion this is not needed, other adapters work well without this.

Reply:
Do you mean the `timeout` or `headers` adjustment? Just removed the timeout of course, but the test has not succeeded without the headers adjustment. I can revert the adjustment if you like and if that works for you, of course. Should I do so?

Reply:
You should not touch the header, it keeps the tests from passing and does not run. Try with just 'headers'

Reply:
Sure, reverted the adjustment, could you recheck it again @Mintiss ? Thanks a lot!